Adds vehicle reid model and updates demo (#1210)
opencv/open_model_zoo: demos/python_demos/multi_camera_multi_target_tracking/README.md

  -u UTILIZATION_MONITORS, --utilization_monitors UTILIZATION_MONITORS
                        Optional. List of monitors to show initially.
```

> Minimum command examples to run the demo for person tracking (for vehicle tracking the commands are the same with appropriate vehicle deetction/reid models):

Reviewer: ```suggestion
Minimum command examples to run the demo for person tracking (for vehicle tracking the commands are the same with appropriate vehicle detection/re-identification models):
```

Author: Fixed

```
# videos
python multi_camera_multi_target_tracking.py \
    -i path/to/video_1.avi path/to/video_2.avi \
    --m_detector path/to/person-detection-retail-0013.xml \
    --m_reid path/to/person-reidentification-retail-0103.xml \

>     --config configs/person.py

Reviewer 2: You should update a default value for `--config`.

Author: Fixed

# videos with instance segmentation model
python multi_camera_multi_person_tracking.py \
    -i path/to/video_1.avi path/to/video_2.avi \
    --m_segmentation path/to/instance-segmentation-security-0050.xml \
    --m_reid path/to/person-reidentification-retail-0107.xml \
    --config configs/person.py

# web-cameras
python multi_camera_multi_person_tracking.py \
    -i 0 1 \
    --m_detector path/to/person-detection-retail-0013.xml \
    --m_reid path/to/person-reidentification-retail-0103.xml \
    --config configs/person.py
```

> The demo can use a JSON file with detections instead of a object detector.

Reviewer: ```suggestion
The demo can use a JSON file with detections instead of an object detector.
```

Author: Fixed

`enable` to `True`, for the second one for `embeddings` specify parameter `path`
that is a directory where data related to embeddings will be saved
(if it is an empty string the option is disabled). In `embeddings` is a parameter

> `use_images`. If it is `True` for every embedding will be drawn an image with a object

Reviewer: ```suggestion
`use_images`. If it is `True` for every embedding will be drawn an image with an object
```

Author: Fixed
